lib/reek/source/source_locator.rb

```diff
@@ -11,60 +12,56 @@ class SourceLocator
       #
       # paths - a list of paths as Strings
       def initialize(paths)
-        @pathnames = paths.
-          map { |path| Pathname.new(path.chomp('/')) }.
-          flat_map { |pathname| current_directory?(pathname) ? pathname.entries : pathname }
+        @paths = paths.map(&method(:Pathname)).flat_map do |path|
```

Comment:
I know i merged the PR, but after thinking about i just came back to say that I believe this is bad. Its valid ruby for sure but incredibly obscure. I mean, quick, how exactly do you call that feature? Right, no idea..;)

WDYT about bejng rather explicit here by just using plain and boring "regular" map?